deployment/terraform/database.tf

```diff
@@ -0,0 +1,104 @@
+#
+# RDS resources
+#
+resource "aws_db_subnet_group" "default" {
+  name        = "${var.rds_database_identifier}"
+  description = "Private subnets for the RDS instances"
+  subnet_ids  = ["${module.vpc.private_subnet_ids}"]
+
+  tags {
+    Name        = "dbsngDatabaseServer"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_db_parameter_group" "default" {
+  name        = "${var.rds_database_identifier}"
+  description = "Parameter group for the RDS instances"
+  family      = "${var.rds_parameter_group_family}"
+
+  parameter {
```

Comment:
I think that we need to make these Terraform variables that get threaded through.